polkadot/xcm/src/v5/mod.rs

```diff
@@ -737,6 +737,15 @@ pub enum Instruction<Call> {
 	/// Errors: None.
 	ClearError,
 
+	/// Set asset claimer for all the trapped assets during the execution.
+	///
+	/// - `location`: The claimer of the assets; it might be an arbitrary location, not necessarily the caller or origin.
```

Comment:
```suggestion
	/// - `location`: The claimer of any assets potentially trapped during the execution of current XCM.
	///   It can be an arbitrary location, not necessarily the caller or origin.
```